src/tests/ftest/pool/list_verbose.py

```diff
@@ -52,17 +52,21 @@ def create_expected(self, pool, scm_free, nvme_free, scm_imbalance,
 
         targets_total = self.server_managers[0].get_config_value("targets") * rank_count
 
+        p_query = self.get_dmg_command().pool_query(pool.identifier)
+        pool_layout_ver = p_query["response"]["pool_layout_ver"]
+        upgrade_layout_ver = p_query["response"]["upgrade_layout_ver"]
```

Comment:
Isn't the point of the `create_expected` method to define the values we expect to see in the dmg pool list/query output?  If so, then doesn't using the the dmg pool query command to produce these expected values undermine the credibility of such a comparison.

Reply:
I initially had the same thought, but I think it's okay here since the test doesn't **really** care about the version. It only checks the version because the entire dictionary returned in the json is compared to the expected values. Rather than only comparing the relevant values

Reply:
Pool version is increasing by various features. Current master pool version is 2, and release 2.2 pool version is 1. Current test script in pool/list_verbose to check pool_layout_ver and upgrade_lay_ver is hard defined with 2 (on master branch) and 1 (on release 2.2).
We need to get a reference (by dmg pool query) for these 2 variables.